src/pytestqt/qtbot.py

```diff
@@ -646,33 +656,15 @@ def mouseRelease(*args, **kwargs):
         qt_api.QtTest.QTest.mouseRelease(*args, **kwargs)
 
     # pep-8 aliases
-
-    def add_widget(self, *args, **kwargs):
-        return self.addWidget(*args, **kwargs)
-
-    def wait_active(self, *args, **kwargs):
-        return self.waitActive(*args, **kwargs)
-
-    def wait_exposed(self, *args, **kwargs):
-        return self.waitExposed(*args, **kwargs)
-
-    def wait_for_window_shown(self, *args, **kwargs):
-        return self.waitForWindowShown(*args, **kwargs)
-
-    def wait_signal(self, *args, **kwargs):
-        return self.waitSignal(*args, **kwargs)
-
-    def wait_signals(self, *args, **kwargs):
-        return self.waitSignals(*args, **kwargs)
-
-    def assert_not_emitted(self, *args, **kwargs):
-        return self.assertNotEmitted(*args, **kwargs)
-
-    def wait_until(self, *args, **kwargs):
-        return self.waitUntil(*args, **kwargs)
-
-    def wait_callback(self, *args, **kwargs):
-        return self.waitCallback(*args, **kwargs)
+    add_widget = addWidget
```

Comment:
Isn't this redundant with what's done in `__init__`?

Reply:
Technically, it is.

The static assignments allow analysis tools to determine that there is an alias and provide auto-completions in IDEs. If the analysis tool can handle the assignment in `__init__()`, the definitions in the class body can be removed.

The re-assignment inside __init__() overwrites these with potential re-implementations in sub-classes.

Reply:
Ok, I tested it in PyCharm and it works there without the definition in the class body.

So should I remove the aliases there and only keep them in `__init__()`?

If yes, what’s your commit policy? Add another commit or squash and force-push?

Reply:
I removed the redundant definitions